applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h

```diff
@@ -0,0 +1,405 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Sergio Jiménez/Alejandro Cornejo/Lucia Barbu
```

Comment:
please remove the accent, it creates problems sometimes (don't remember exactly)
```suggestion
//  Main authors:    Sergio Jimenez/Alejandro Cornejo/Lucia Barbu
```